machine-learning/core-ml-intern.md

```diff
@@ -0,0 +1,47 @@
+---
+authors: @sanchit
+groups: product
+teams: core-ml
+level: NA
+active: true
+---
+# Machine Learning Research Intern
+
+We are looking for an ML Research intern for a duration of 5 months. As an intern, you will be working closely with the core-ml team. You will be involved in owning various product initiatives, creating protypes and delving into research papers. 
+Our team works closely with the research team, broadly on the topics such as Spoken Language Understanding, Speech Recognition, Paralinguistics and Response Production.
+
+### Responsibilities
+
++ Design, build and evaluate Machine Learning systems.
++ Perform experiments and statistical analyses to draw conclusions and take
+  modeling decisions.
++ Study, implement and extend state of the art systems.
++ Write research outcomes and publish documentations
```

Comment:
We can remove this part. This intern will be more focused on product development work.